dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryLoggingExporterTest.java

```diff
@@ -0,0 +1,80 @@
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal_fat;
+
+import static com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions.SERVER_ONLY;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.annotation.TestServlets;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import io.openliberty.microprofile.telemetry.internal_fat.apps.telemetry.ConfigServlet;
+
+//@Mode(TestMode.FULL)
+@RunWith(FATRunner.class)
+public class TelemetryLoggingExporterTest extends FATServletClient {
+
+    public static final String SERVER_NAME = "Telemetry10ConfigEnv";
+    public static final String APP_NAME = "TelemetryApp";
+
+    @Server(SERVER_NAME)
+    @TestServlets({
+                    @TestServlet(servlet = ConfigServlet.class, contextRoot = APP_NAME),
+    })
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        WebArchive app = ShrinkWrap.create(WebArchive.class, APP_NAME + ".war")
+                        .addClasses(ConfigServlet.class);
+
+        ShrinkHelper.exportAppToServer(server, app, SERVER_ONLY);
+
+        server.addEnvVar("otel_service_name", "overrideDone");
+        server.addEnvVar("otel_sdk_disabled", "false");
+
+        //Use logging exporter
+        server.addEnvVar("otel_traces_exporter", "logging");
+        server.startServer();
+    }
+
+    @Test
+    public void testConsole() throws Exception {
+        runTest(server, APP_NAME + "/ConfigServlet", "testServiceNameConfig");
+
+        //The exporter should not give this error message
+        assertTrue(server.findStringsInLogs("Failed to export spans").isEmpty());
+
+        //Checks for span output in logs
+        assertFalse(server.findStringsInLogs("io.opentelemetry.exporter.logging.LoggingSpanExporter").isEmpty());
+        assertFalse(server.findStringsInLogs("'testSpan' : .* \\[tracer: config-test:1.0.0\\]").isEmpty());
```

Comment:
Unless you can guarantee that these strings must be in the logs at this point, consider using `server.waitForStringsInLogs` instead. I think you can get a slight delay between when a span is created and when it's exported, I think you want to do that here to avoid intermittent defects.

Also, if you need to ensure that a particular operation outputs a certain string, you can use `server.setMarkToEndOfLog` first, then perform the operation under test, and then call `server.waitForStringInLogUsingMark` afterwards.